packages/react-components/react-tabster/src/hooks/useModalAttributes.ts

```diff
@@ -30,6 +33,19 @@ export interface UseModalAttributesOptions {
   id?: string;
 }
 
+/**
+ * !!DANGEROUS!! Designates an element that will not be hidden even when outside an open modal.
+ * Only works for top-level elements; should be used with extreme care.
+ * @returns Attribute to apply to the target element that should never receive aria-hidden
+ */
+export function useDangerousNeverHidden_unstable(): { [key: string]: string } {
+  return React.useRef({ [DangerousNeverHiddenAttribute]: '' }).current;
```

Comment:
Is it just a regular const? I.e.:

```
const DangerousNeverHiddenAttr = { [DangerousNeverHiddenAttribute]: '' }
```

Reply:
oh yeah, it does make more sense to pull that out of the hook, I'll define it up where `DangerousNeverHiddenAttribute` is now